### PR TITLE
Create meta.yml for traja

### PR DESCRIPTION
**Pull request by justinshenk**

### CODE REVIEW

**Comment by justinshenk** [comment]
@conda-forge-coordinator Pull request has not led to `traja-feedstock` creation. Can you please help?

**Comment by kmuehlbauer** [comment]
@justinshenk It isn't merged yet. ping @conda-forge/help-python

File: recipes/traja/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d13d54d66887b0c2de1db6d5161d72e96f8411364364e6b442e7e5f341f09a5f
+
+build:
+  number: 1
```
Comment (chrisburr) [inline recipes/traja/meta.yaml]: Build numbers should start at 0 for new versions

File: recipes/traja/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
+{% set name = "traja" %}
+{% set version = "0.0.2a2" %}
```
Comment (chrisburr) [inline recipes/traja/meta.yaml]: Pre release (alpha/beta) versions are normally not added to `conda-forge`.

I see there is a `0.0.4` release on PyPI and a `0.0.7` release on GitHub, could you update it?

File: recipes/traja/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+test:
+  imports:
+    - traja
+    - traja.contrib
+
+about:
+  home: https://github.com/justinshenk/traja
+  license: MIT
+  license_family: MIT
+  license_file:
```
Comment (chrisburr) [inline recipes/traja/meta.yaml]: This should specify the name of the license file in the repository:
```suggestion
  license_file: LICENSE
```

Comment (justinshenk) [reply recipes/traja/meta.yaml]: Thanks, added!

File: recipes/traja/meta.yaml (hunk 2 above)
Comment (justinshenk) [reply recipes/traja/meta.yaml]: Updated PyPI to `0.0.7`, thank you.

Comment (chrisburr) [reply recipes/traja/meta.yaml]: You also need to update the hash of the tarball, see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L19-L22).

Comment (justinshenk) [reply recipes/traja/meta.yaml]: Updated, thanks!

File: recipes/traja/meta.yaml (hunk 1 above)
Comment (justinshenk) [reply recipes/traja/meta.yaml]: Done, thanks

**Comment by justinshenk** [comment]
@chrisburr It looks ready to merge. If anything is still missing please let me know.

File: recipes/traja/LICENSE [hunk 4]
```diff
@@ -0,0 +1,7 @@
+Copyright 2019 Justin Shenk
+
+Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
```
Comment (chrisburr) [inline recipes/traja/LICENSE]: (This doesn't prevent this from being merged but so you know for future releases.)

This file shouldn't needed as, according to this line, there should be a copy of the MIT licence included with every distribution of the package. It should be automatically included in the release tarballs on PyPI if you edit your `setup.cfg` to include:

```
[metadata]
license_file = LICENSE
```

File: recipes/traja/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 873c671ccb48a79b1e6e48c70ab9b43819f96241bb339aabacc1ed2e6afd3896
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  skip: true  # [win]
```
[the remark this answers is not in the document]
Comment (justinshenk) [reply recipes/traja/meta.yaml]: Yes, you are correct. Thank you for the information. Reverting to the original.

File: recipes/traja/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 873c671ccb48a79b1e6e48c70ab9b43819f96241bb339aabacc1ed2e6afd3896
+
+build:
```
[the remark this answers is not in the document]
Comment (justinshenk) [reply recipes/traja/meta.yaml]: Done.

File: recipes/traja/LICENSE (hunk 4 above)
Comment (justinshenk) [reply recipes/traja/LICENSE]: That's just the information I was looking for, thank you!

**Comment by chrisburr** [comment]
@conda-forge/staged-recipes I think this is ready to merge

File: recipes/traja/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  noarch: python
+
+requirements:
+  host:
+    - pip
+    - python>=3
+  run:
+    - python>=3
```
Comment (beckermr) [inline recipes/traja/meta.yaml]: We want a space between python and `>=3`

Comment (justinshenk) [reply recipes/traja/meta.yaml]: Space added with e22fe13. Ready for merging.

**Pull request merged**